Assets/Scripts/CountdownController.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class CountdownController : MonoBehaviour
+{
+    public int countdownTime;
```

Comment:
Please make your fields private and add the SerializeField attribute so they can be modified in the editor instead of making your fields public.